src/bootstrap/flags.rs

```diff
@@ -430,14 +423,9 @@ Arguments:
     This subcommand accepts a number of paths to directories to the crates
     and/or artifacts to compile. For example:
 
-        ./x.py check library/core
-        ./x.py check library/core library/proc_macro
+        ./x.py check library/std
 
-    If no arguments are passed then the complete artifacts are compiled: std, test, and rustc. Note
-    also that since we use `cargo check`, by default this will automatically enable incremental
-    compilation, so there's no need to pass it separately, though it won't hurt. We also completely
-    ignore the stage passed, as there's no way to compile in non-stage 0 without actually building
-    the compiler.",
+    If no arguments are passed then all artifacts are checked.",
```

Comment:
```suggestion
    If no arguments are passed then many artifacts are checked.",
```

Currently we don't support x.py check for *everything*, which means that it's actually somewhat more limited than this might indicate on the surface. (We should eventually likely expand the support, but haven't done that yet).

Reply:
Done.